src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -138,6 +138,8 @@ def __init__(self, config):
         # These are added now by the condor client
         #self.x509userproxysubject = proxy.getSubject()
         #self.x509userproxyfqan = proxy.getAttributeFromProxy(self.x509userproxy)
+        
+        self.tc = TagCollector()
```

Comment:
I would suggest initializing this class with `cacheduration` of 10min, or something like this.

Reply:
@amaltaro The default right now is 1 hour, and is what ReqMgr uses. Do you think that is too long?

Reply:
Naah, I think 1h is pretty good! Honestly, I am always confused if that cacheduration is in seconds, minutes or hours (I thought it was set to 1 second :))